documentation/src/main/asciidoc/server_guide/configuration.adoc

```diff
@@ -498,3 +498,36 @@ and editing the module definition to include the jar as resource-root:
 
 ----
 
+==== Custom Marshaller Bridges
+Infinispan provides two marshalling bridges for marshalling client/server requests using the Kryo and Protostuff libraries.
+To utilise either of these marshallers, you simply place the dependency of the marshaller you require in your client
+pom. e.g. for Protostuff:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.infinispan</groupId>
+  <artifactId>infinispan-marshaller-protostuff</artifactId>
+  <version>{infinispanversion}</version>
+</dependency>
+----
+
+Custom schemas for object marshalling must then be registered with the selected library using the libraries api on the client.
```

Comment:
Are you building a tutorial or example of this? As a user I'd like to see a full example. This could be as a simple tutorial, or by adding an example in the documentation itself.